apps/admin-web/src/app/app.tsx

```diff
@@ -116,26 +119,14 @@ export const App = () => {
           fullWidth
           id="outlined-search"
           type="search"
-          label="Filter"
+          label="Search"
           variant="outlined"
-          onChange={filter}
+          onChange={debounce(sort, 200)}
```

Comment:
Uses lodash's `debounce` since our sort function can be expensive in an onchange. onChange will run on each keystroke and I was starting to notice a slight performance issue with the little mocked routes we have. Debounce will wait 200ms after the last onchance is fired before running.

I picked 200ms because 300ms has a slight delay. The average reaction time for a human is 250ms https://www.pubnub.com/blog/how-fast-is-realtime-human-perception-and-technology/.

Reply:
Taking one step back old way had no performance impact on key press. Does this implementation of fuzzy search seem better? I'm fine trying but with endpoints I feel like filtering exact may be fine